src/main/java/lotto/domain/LastWeekWinnerNumber.java

```diff
@@ -2,12 +2,29 @@
 
 public class LastWeekWinnerNumber {
 	private final Lotto winnerNumber;
+	private final LottoNumber bonusNumber;
 
-	public LastWeekWinnerNumber(String inputNumber) {
-		winnerNumber = new Lotto(inputNumber);
+	public LastWeekWinnerNumber(String inputNumber, LottoNumber bonusNumber) {
+		this.winnerNumber = new Lotto(inputNumber);
+		if (this.winnerNumber.getGameNumberList().contains(bonusNumber)) {
```

Comment:
getter를 통해 내부 정보를 외부에 노출하지 말고 Lotto에게 메시지를 보내 직접 물어보면 어떨까요~?
```suggestion
		if (winnerNumber.hasContain(bonusNumber)) {
```